playground/terraform/applications/backend-python/main.tf

```diff
@@ -0,0 +1,45 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+resource "google_app_engine_flexible_app_version" "backend_app_python" {
+  version_id = "v1"
+  project    = "${var.project_id}"
+  service    = "${var.service_name}"
+  runtime    = "custom"
+  instance_class = "B4"
+
+ liveness_check {
+    path = ""
+  }
+
+  readiness_check {
+    path = ""
+  }
+
+  manual_scaling {
```

Comment:
Use automatic_scaling and max_total_instances?
There is also min_idle_instances - so we can keep some minimal idle and warmed-up capacity to improve responsiveness?

Reply:
@snkalinin can you address these questions?